README.md

```diff
@@ -420,6 +420,10 @@ Last but not least, please note that commands for SSR are different to the ones
 - Use `sku build-ssr` to build your production assets. You can then run `node ./dist/server.js`. Your server will run at `http://localhost:xxxx`, where `xxxx` is `serverPort`.
 - Use `sku test-ssr` to test your application
 
+## Bundle Analysis
```

Comment:
Maybe reference the actual plugin with a link. Also, can we move this to the `features` section?

Reply:
Done.